### ISSUE TITLE
Accessibility bug in the Contact facilities Page within the cheetah MVP flow

### ISSUE THREAD

**Issue by jenniemc** [department-of-veterans-affairs/va.gov-team#21842]
Discovered in #21342

**Contact facilities page**: 
1. Wrong heading order
-  Select `Yes` on the [Have you received a Covid-19 vaccine](http://localhost:3001/health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/received-dose) page 
-  On the [contact-facilities](http://localhost:3001/health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/contact-facilities), the order of the heading is incorrect. It currently reads: Your facilities (h2), Rawlins VA Clinic (h2), main phone (h3). 
- Expected behavior is that the clinic names are subheading to `Your facilities`

2. Invalid ARIA attribute value: aria-labelledby="vaos-facilities-label"
- It is missing the `id="vaos-facilities-label"` 
- Expected behavior is the aria-labelledby can be reference by the a tag with `id="vaos-facilities-label"`
- See secondary Contact facility scenario (when user has no facility available for covid-19 vaccine) as a valid ARIA example. 
- To get "no facility" event, go into `src/applications/vaos/services/mocks/var/direct_booking_eligibility_criteria.json` and remove the `patientHistoryRequired` property from all the `covid` types of care

![image.png](https://images.zenhubusercontent.com/5d6463ed1325bafe0674800b/9876dfc3-3d73-41b8-8781-604d80338a6b)

3. Add Cypress test to cover the two Contact facilities scenario: When user selects `Yes` to Have you received a Covid-19 vaccine page and when user has no registered facility available for covid-19 vaccine.

**Comment by jenniemc** [comment]
@laurenernest, I don't think this is a design problem but is a coding issue that the FE can fix.

**Comment by laurenernest** [comment]
We need to validate the heading structure against the specs to make sure we have it updated everywhere (and doesn't become a repeat issue if we touch this page again)

**Comment by laurenernest** [comment]
@jenniemc Is the list of facilities shared between the different versions of the page (none online, dose received)? Or can they be coded as different heading levels depending on the other conditions and content of the same page (/health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/contact-facilities)?

![Screen Shot 2021-03-24 at 3.51.56 PM.jpg](https://images.zenhubusercontent.com/5f7dd2dad6b9a9301bbab94e/4c3da947-e2b9-4639-908e-7b1a2968b880)

**Comment by outerpress** [comment]
@jenniemc @laurenernest adding the "Your facilities" header to the no-online version works for me.
